scripts/asm-installer/asm_vm

```diff
@@ -946,7 +946,15 @@ uses_environ_workload_identity() {
 
   if [[ "$(retry 2 kubectl get configmap "${CONFIGMAP}" -n istio-system \
     -ojsonpath='{.data.mesh}' | grep -x -c "^trustDomain: ${PROJECT_ID}.hub.id.goog$" \
```

Comment:
We can achieve the check in one line with an OR in the regex. e.g. https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages/blob/master/scripts/asm-installer/install_asm#L2201

Reply:
Done. Please take a look. Thanks!

Reply:
LGTM